### ISSUE TITLE
Options flag doesnt work

### ISSUE THREAD

**Issue by ep4sh** [rianhunter/dbxfs#55]
Hey guys, just want to share my issue with dbxfs 
```
root@master:/home/ubuntu/ehome/app# ls -lai /dropbox/
total 8
791310 drwxr-xr-x  2 root root 4096 Jan  2 15:53 .
     2 drwxr-xr-x 21 root root 4096 Jan  2 15:53 ..
root@master:/home/ubuntu/ehome/app# dbxfs -v -o allow_other /dropbox
2021-01-02 16:26:32,139:INFO:dropbox:Request to users/get_current_account
2021-01-02 16:26:32,751:INFO:dbxfs.main:Starting dbxfs...
Traceback (most recent call last):
  File "/usr/local/bin/dbxfs", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/dbxfs/main.py", line 496, in main
    return _main(argv)
  File "/usr/local/lib/python3.8/dist-packages/dbxfs/main.py", line 485, in _main
    return userspacefs.simple_main(mount_point, display_name,
  File "/usr/local/lib/python3.8/dist-packages/userspacefs/__init__.py", line 404, in simple_main
    return mount_and_run_fs(display_name, create_fs_params,
  File "/usr/local/lib/python3.8/dist-packages/userspacefs/__init__.py", line 352, in mount_and_run_fs
    os.environ['__userspacefs_fuse_opt_' + key] = value
  File "/usr/lib/python3.8/os.py", line 680, in __setitem__
    value = self.encodevalue(value)
  File "/usr/lib/python3.8/os.py", line 750, in encode
    raise TypeError("str expected, not %s" % type(value).__name__)
TypeError: str expected, not bool
Sentry is attempting to send 0 pending error messages
Waiting up to 2 seconds
Press Ctrl-C to quit
```
Can you please suggest what I do wrong? Thanks in advance

**Comment by rianhunter** [comment]
This is a simple bug, you should be able to fix it if you have a basic knowledge of python otherwise you'll have to wait for a fix. If you want me to prioritize this, please send 25 mBTC to 1Am6AZHzrRvejYR3wBjo3oG4xUnbCa51dM and I'll fix it the next free moment I have.

**Comment by rianhunter** [comment]
Fixed in 8b1254dc8123590c4cac171ca711a8de7a80ca8c
